webdrivers.gemspec

```diff
@@ -19,6 +19,8 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
   s.require_paths = ['lib']
 
+  s.required_ruby_version = '>= 2.3.0'
```

Comment:
Ci has been failed.

```console
Offenses:
webdrivers.gemspec:22:29: C: Gemspec/RequiredRubyVersion: required_ruby_version (2.3, declared in webdrivers.gemspec) and TargetRubyVersion (2.4, which may be specified in .rubocop.yml) should be equal.
  s.required_ruby_version = '>= 2.3.0'
                            ^^^^^^^^^^
30 files inspected, 1 offense detected
```

https://travis-ci.org/titusfortner/webdrivers/jobs/595017355#L482-L493

It seems to need to be specified with `>= 2.4.0`.

```diff
-s.required_ruby_version = '>= 2.3.0'
+s.required_ruby_version = '>= 2.4.0'
```